config/routes.rb

```diff
@@ -3,11 +3,11 @@
   post '/login', to: 'sessions#create'
   get '/logout', to: 'sessions#destroy'
 
-  resources :events, only: [:show]
-
   namespace :admin do
     get '/dashboard', to: 'base#dashboard'
     resources :announcements
   end
-  resources :dashboard
+
+  get '/dashboard', to: 'dashboard#index'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.